### DISCUSSION TITLE
Change Output Template

### DISCUSSION THREAD

**Discussion by sirlaughalot90** [chrisgrieser/pdf-annotation-extractor-alfred#21] [Q&A] [answered]
Hi,

Is it possible to change the output template? I wasn't able to find anything in the documentation (maybe I'm blind, I dunno. 😄). Thanks!

**Comment by chrisgrieser** [comment, marked as answer]
Hi,

Changing output template is currently not possible. If you could file a Feature Request and there is also interest for that by other people, I'll look into it
